cocotb/setup.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
```

Comment:
Duplicate file, it looks like half your changes are in this one, and half in the other setup.py. Can you merge them into the other one?